source/common/upstream/edf_scheduler.h

```diff
@@ -89,7 +107,8 @@ template <class C> class EdfScheduler {
     }
   };
 
-  // Current time in EDF scheduler.
+  // Current time in EDF scheduler. It could be initiated as a float ranges [0.0, 1.0) to avoid
+  // biasing towards earlier hosts in the schedule across refreshes for the weighted case.
```

Comment:
Can you reword this to make it a bit clearer as to why the initializing to the range avoids bias?